src/@next/Badge/Badge.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Typography } from '../Typography';
+import { Neutral } from '../utilities/colors';
+import { BadgeStyle } from './BadgeStyle';
+
+export type BadgeStatusType =
```

Comment:
can you replace this with https://github.com/glints-dev/glints-aries/pull/709/files#diff-938d02609f5a10f4d8c5ee533ad0b79fb3284622aa8603ef9b12d4d0a027413dR7-R15 (and remove the one from BadgeStyle)

and `export type BadgeStatusVariant = typeof BadgeStatusVariant[number];`

so both the component and the mapping refer the same type.